guides/login-with-github.mdx

```diff
@@ -0,0 +1,116 @@
+## Login With GitHub
+
+If you've read the [Login With Google guide](/guides/login-with-google), you would have learned that Flipt supports many methods of authentication for users to control who has access to what in Flipt.
+
+Alongside the support for OIDC Login that supports many providers, Flipt has launched support for login with GitHub in version [v1.26.0](https://github.com/flipt-io/flipt/tree/v1.26.0), through the OAuth2.0 flow. This guide will serve as a walk-through on how to set this flow up for users of Flipt in your organization.
+
+## Prerequisites
+
+- [Docker](https://www.docker.com/)
+- [GitHub](https://github.com/)
+- [Ngrok](https://ngrok.com/)
+
+## Brief on OAuth2.0
+
+OAuth2.0 is an authentication standard whose goal is to allow 3rd party applications to access authorized resources from a provider. It relies on the user explicitly granting access to the 3rd party application to issue a token on behalf of the OAuth2.0 provider for authorized use.
+
+![OAuth2.0 Flow Diagram](/images/guides/login-with-github/oauth-flow-diagram.png)
```

Comment:
I wonder if we could make a similar diagram using something like d2 or mermaid. Just because this is auth0 specific and it adds an extra actor to the equation (separation of auth0 and "your api"), which isn't really valid in this instance.

Reply:
Copied this from: https://www.digitalocean.com/community/tutorials/an-introduction-to-oauth-2

Wanted to have a very general diagram, not related to Flipt or anything else. There are different OAuth flows as well, so wanted to keep it as general as possible.